.circleci/config.yml

```diff
@@ -188,6 +188,10 @@ jobs:
           paths:
             - ~/workbench/ui/node_modules
           key: ui-cache-{{ checksum "~/workbench/ui/package.json" }}
+      - run:
+          name: Lint Angular app
+          working_directory: ~/workbench/ui
+          command: yarn lint
```

Comment:
The command would be `yarn run lint`

Reply:
Done for consistency with `./project.rb`. FTR `yarn lint` appears to have done the same thing.